glotaran/builtin/megacomplexes/spectral/spectral_megacomplex.py

```diff
@@ -61,18 +68,23 @@ def finalize_data(self, dataset_model: DatasetModel, data: xr.Dataset):
                     compartment for compartment in megacomplex.shape if compartment not in species
                 ]
 
-        data.coords["species"] = species
+        data.coords[species_dimension] = species
+        matrix = data.global_matrix if as_global else data.matrix
```

Comment:
This lines currently crashes example [fit_simData05.py](https://github.com/glotaran/pyglotaran-validation/blob/main/paramGUI/simData05/fit_simData05.py)

Reply:
Thats fixed